hadoop-ozone/dist/src/main/compose/ozone-grpc-om-s3/README.md

```diff
@@ -0,0 +1,26 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+# Using gRPC between OM and S3G
+
+This is the same as `compose/ozone` but for testing operations with gRPC enabled 
```

Comment:
Did we already enable gRPC between S3G and OM by default? If so, do we really need to have this as a separated docker-compose setup, or just make gRPC enabled and tested by default as per `compose/ozone`?

Reply:
There is an open PR for enabling it, but it seems to take longer than expected. Therefore, I created a separated docker setup. If the PR gets merged we can get rid of it. But I don't know if that will be soon.

Reply:
> do we really need to have this as a separated docker-compose setup

Not necessary, we can modify the existing dev cluster to only run our acceptance test when the cluster is configured to use s3g Grpc, specifically the `om.transport.class = org.apache.hadoop.ozone.om.protocolPB.GrpcOmTransportFactory`. 
The acceptance test for s3g metrics can be inserted into the existing dev clusters such as `compose/ozone`.  Within the test, it needs to check if the `ozone-site` is configured for s3g Grpc such as,
```
   ${result} =      Execute And Ignore Error    ozone getconf confKey om.transport.class
   ${contains} =    Evaluate        "GrpcOmTransport" in """${result}"""
   IF   ${contains} == ${True}
```
run test.  If the config is not set, then either do something else or do nothing.  
As @xBis7 mentioned, once the open PR to enable Grpc by default is merged and closed, the test will be run by default.  Until then, it will be run when the user configures the dev cluster to use the `GrpcOmTransport`.

Reply:
I see, thanks for the detailed explanation.

Reply:
Thanks @neils-dev for the robot code to check for grpc conf. I made the changes.